orc8r/cloud/deploy/terraform/orc8r-aws/eks.tf

```diff
@@ -49,7 +49,11 @@ module "eks" {
   }
   worker_additional_security_group_ids = concat([aws_security_group.default.id], var.eks_worker_additional_sg_ids)
   workers_additional_policies          = var.eks_worker_additional_policy_arns
-  worker_groups                        = var.thanos_enabled ? concat(var.eks_worker_groups, var.thanos_worker_groups) : var.eks_worker_groups
+  worker_groups                        = (
```

Comment:
What's this line doing? Is a bit hard to follow

Reply:
The idea is to substitute the existing `var.eks_worker_groups` by a conditional check based on the `enable_orc8r_blue_green_deployment` flag.

Because there's already a conditional check I had to nest another one. Though thinking now, I can make it more readable using a local variable, something like:

```
locals {
  orc8r_worker_group = var.enable_orc8r_blue_green_deployment ? var.blue_green_worker_groups : var.eks_worker_groups
}

...

worker_groups = var.thanos_enabled ? concat(local.orc8r_worker_group, var.thanos_worker_groups) : local.orc8r_worker_group
```